pkg/cmd/codespace/list.go

```diff
@@ -53,10 +54,20 @@ func newListCmd(app *App) *cobra.Command {
 	listCmd.Flags().StringVarP(&opts.userName, "user", "u", "", "The `username` to list codespaces for (used with --org)")
 	cmdutil.AddJSONFlags(listCmd, &exporter, api.CodespaceFields)
 
+	listCmd.Flags().BoolVarP(&opts.useWeb, "web", "w", false, "List codespaces in the web browser")
```

Comment:
Can this text point out the exclusivity with `--user` and `--org`?

Reply:
fixed in [c4228f1](https://github.com/cli/cli/pull/7288/commits/c4228f11cb2f328dd4afe577c0ce1167e0b30dbe) .
I change it to : `"List codespaces in the web browser, can only be used with --repo flag"`
This commit also clarify other issue like:
- removing 'yet' in error messages